www/src/pages/blog/tags.js

```diff
@@ -39,11 +35,8 @@ const POPULAR_TAGS = [
 const PopularTagGrid = styled.div`
```

Comment:
Could you add a comment here with a link to the article? Something like:

```suggestion
// Responsive tag list using grid:
// https://css-tricks.com/look-ma-no-media-queries-responsive-layouts-using-css-grid/
const PopularTagGrid = styled.div`
```

Reply:
@tesseralis I have added the comment